playbooks/roles/ocp-customization/README.md

```diff
@@ -20,6 +24,12 @@ Role Variables
 |-------------------------|----------|----------------|---------------------------------------------|
 | workdir                 | no       | ~/ocp4-workdir | Place for config generation and auth files  |
 | rhcos_kernel_options    | no       | []             | List of kernel options for RHCOS nodes eg: ["slub_max_order=0","loglevel=7"] |
+| sysctl_tuned_options    | no       | false       | Set to true to apply sysctl options via tuned operator |
+
+
+`tuned operator options`
```

Comment:
Missed something here? This messed up the table.

Reply:
hmm..

Reply:
fixed